en/docs/consume/discover-apis/search.md

```diff
@@ -16,9 +16,17 @@ You can search for APIs in the API Publisher or Developer Portal in the followin
 </tr>
 </thead>
 <tbody>
+<tr class="even">
+<td>Full text search</td>
+<td><p>As this is the default option, simply enter any attribute value 
+of the API and search.</p>
+<p>For example, PizzaShackAPI will search for PizzaShackAPI in API name, context, version, documents, etc.</p>
+</td>
+</tr>
 <tr class="odd">
 <td>By the API's name</td>
-<td>As this is the default option, simply enter the API's name and search.</td>
+<td><p><strong>name:xxxx</strong> . For example, name:PizzaShackAPI</p>
+<p>Provider is the user who created the API.</p></td>
```

Comment:
I think this line
```
Provider is the user who created the API.
```
 is not valid here

Description for search by the provider is there right below the name search 🌞 
![image](https://user-images.githubusercontent.com/3313885/102850622-e3c68a00-443f-11eb-964b-c7c1ae2ab4b3.png)